src/net_info.js

```diff
@@ -5,16 +5,19 @@
    * - online - when browser goes online
    * - offline - when browser goes offline
    */
+
+  var _window = Pusher.runtime.getWindow();
```

Comment:
Net info feels like it should be runtime specific. Certainly the way this information will be detected will be runtime dependent.